k8s/nginx/staging/platform-vs.yaml

```
@@ -309,9 +309,13 @@ spec:
               - name: Content-Type
                 value: application/json
     - path: ~ /api\/v[1-2]\/beacon
+      action:
+        pass: beacon                
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)